concepticondata/conceptlists/Johansson-2020-344.tsv

```diff
@@ -0,0 +1,345 @@
+Johansson-2020-344-257	257	WHAT?	1236	WHAT
+Johansson-2020-344-258	258	WHEN?	1238	WHEN
+Johansson-2020-344-259	259	WHERE?	1237	WHERE
+Johansson-2020-344-260	260	WHITE	1335	WHITE
+Johansson-2020-344-261	261	WHO?	1235	WHO
+Johansson-2020-344-262	262	WHY?	1270	WHY
+Johansson-2020-344-263	263	WIDE	1243	WIDE
+Johansson-2020-344-264	264	WING	1257	WING
+Johansson-2020-344-265	265	WIND	960	WIND
+Johansson-2020-344-266	266	WOMAN	962	WOMAN
+Johansson-2020-344-267	267	WORD	1599	WORD
+Johansson-2020-344-268	268	WRONG	1390	WRONG
+Johansson-2020-344-269	269	YAWN	1028	YAWN
+Johansson-2020-344-270	270	YEAR	1226	YEAR
+Johansson-2020-344-271	271	YELLOW	1424	YELLOW
+Johansson-2020-344-272	272	YESTERDAY	1174	YESTERDAY
+Johansson-2020-344-273	273	YOUNG	1207	YOUNG
+Johansson-2020-344-274	274	1SG	1209	I
+Johansson-2020-344-275	275	1PLI	1131	WE (INCLUSIVE)
+Johansson-2020-344-276	276	1PLE	1130	WE (EXCLUSIVE)
+Johansson-2020-344-277	277	2SG	1215	THOU
+Johansson-2020-344-278	278	2PL	1213	YOU
+Johansson-2020-344-279	279	3SG	262	HE OR SHE OR IT
+Johansson-2020-344-280	280	3PL	817	THEY
+Johansson-2020-344-281	281	mother's mother (female speaking)		
+Johansson-2020-344-282	282	mother's mother (male speaking)		
+Johansson-2020-344-283	283	father's mother (female speaking)		
+Johansson-2020-344-284	284	father's mother (male speaking)		
+Johansson-2020-344-285	285	mother's father (female speaking)		
+Johansson-2020-344-286	286	mother's father (male speaking)		
+Johansson-2020-344-287	287	father's father (female speaking)		
+Johansson-2020-344-288	288	father's father (male speaking)		
+Johansson-2020-344-289	289	mother (female speaking)		
+Johansson-2020-344-290	290	mother (male speaking)		
+Johansson-2020-344-291	291	father (female speaking)		
+Johansson-2020-344-292	292	father (male speaking)		
+Johansson-2020-344-293	293	mother's older sister (female speaking)		
+Johansson-2020-344-294	294	mother's older sister (male speaking)		
+Johansson-2020-344-295	295	mother's younger sister (female speaking)		
+Johansson-2020-344-296	296	mother's younger sister (male speaking)		
+Johansson-2020-344-297	297	father's older sister (female speaking)		
+Johansson-2020-344-298	298	father's older sister (male speaking)		
+Johansson-2020-344-299	299	father's younger sister (female speaking)		
+Johansson-2020-344-300	300	father's younger sister (male speaking)		
+Johansson-2020-344-301	301	mother's older brother (female speaking)		
+Johansson-2020-344-302	302	mother's older brother (male speaking)		
+Johansson-2020-344-303	303	mother's younger brother (female speaking)		
+Johansson-2020-344-304	304	mother's younger brother (male speaking)		
+Johansson-2020-344-305	305	father's older brother (female speaking)		
+Johansson-2020-344-306	306	father's older brother (male speaking)		
+Johansson-2020-344-307	307	father's younger brother (female speaking)		
+Johansson-2020-344-308	308	father's younger brother (male speaking)		
+Johansson-2020-344-309	309	older sister (female speaking)		
```

Comment:
For this section of the kinship terms, there are some concept sets in Concepticon, e.g. OLDER BROTHER (OF MAN), OLDER BROTHER (OF WOMAN). Or does "female speaking/male speaking" imply something else than "of female ego/of male ego"?

Reply:
I've checked now; male or female speaking means the male or female ego. So you're right, there are a couple of kinship terms that can actually be mapped. Thanks for the pointer!